includes/class-wp-smime.php

```diff
@@ -141,9 +159,15 @@ public static function encrypt ( $message, $headers, $certificates ) {
 
         if ( $smime ) {
             $parts = explode( "\n\n", $smime, 2 );
+
+            // add new mime type for improved compatibility (e.g. Roundcube)
+            // https://tools.ietf.org/html/rfc3851
+            if ( false === stripos( $parts[0], "application/pkcs7-mime" ) ) {
+                $parts[0] .= "\nContent-Type: application/pkcs7-mime;\n";
+            }
             $r = array(
                 'headers' => $parts[0],
-                'message' => $parts[1],
```

Comment:
It doesn't particular matter but just FYI, this comma can stay. Unlike JSON, PHP arrays can have commas after their last element, which helps to avoid syntax errors if the elements are ever shuffled in a copy/paste operation.